docs/source/upcoming_release_notes/859-group_device.rst

```diff
@@ -0,0 +1,91 @@
+859 group_device
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- Add GroupDevice: A device that is a group of components that will act independently.
+
+Device Updates
+--------------
+- The following devices have become group devices:
+
+  - Acromag
```

Comment:
Nitpick: sort by name?

Reply:
You're right. I'm just being a bit lazy at the end of a day, should have sorted to begin with.